content/en/security/application_security/code_security/setup/python.md

```diff
@@ -106,6 +106,22 @@ Update your ECS task definition JSON file, by adding this in the environment sec
 If you need additional assistance, contact [Datadog support][5].
 
 
+### Third-Party library compatibility note
+
+Code Security modifies Python code at runtime, which can cause conflicts with other third-party Python libraries that perform similar code transformations, especially but not limited to:
```

Comment:
```suggestion
Code Security modifies Python code at runtime. This could cause conflicts with other third-party Python libraries that perform similar code transformations, particularly with the following, though not limited to them:
```